autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -93,6 +93,26 @@ export function parseSections(arr: any[]): SectionSelected[] {
   return sections;
 }
 
+function sortSections(sections: SectionSelected[]): SectionSelected[] {
```

Comment:
I'd add doc comments to this describing the general sorting idea (i.e. grouping, sort groups by smallest section number, TBA sections last)